<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWdgt</class>
 <widget class="QMainWindow" name="MainWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>494</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowIcon">
   <iconset resource="mnemosyne.qrc">
    <normaloff>:/mnemosyne/pixmaps/mnemosyne.png</normaloff>:/mnemosyne/pixmaps/mnemosyne.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>315</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionManagePlugins"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionGettingStarted"/>
    <addaction name="actionTip"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionSync"/>
    <addaction name="actionCompact"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Cards">
    <property name="title">
     <string>&amp;Cards</string>
    </property>
    <addaction name="actionAddCards"/>
    <addaction name="actionEditCurrentCard"/>
    <addaction name="actionDeleteCurrentCard"/>
    <addaction name="separator"/>
    <addaction name="actionBrowseCards"/>
    <addaction name="actionActivateCards"/>
    <addaction name="actionFindDuplicates"/>
    <addaction name="actionManageTypes"/>
    <addaction name="separator"/>
    <addaction name="actionShowStatistics"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Cards"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddCards"/>
   <addaction name="actionEditCurrentCard"/>
   <addaction name="actionDeleteCurrentCard"/>
   <addaction name="actionBrowseCards"/>
   <addaction name="actionShowStatistics"/>
  </widget>
  <action name="actionActivateCards">
   <property name="text">
    <string>(De)a&amp;ctivate cards...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAddCards">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/add.png</normaloff>:/mnemosyne/pixmaps/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add cards...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/configure.png</normaloff>:/mnemosyne/pixmaps/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure Mnemosyne...</string>
   </property>
  </action>
  <action name="actionDeleteCurrentCard">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/delete.png</normaloff>:/mnemosyne/pixmaps/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete current card</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEditCurrentCard">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/edit.png</normaloff>:/mnemosyne/pixmaps/edit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit current card...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionBrowseCards">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/cards.png</normaloff>:/mnemosyne/pixmaps/cards.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Browse cards...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export...</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/exit.png</normaloff>:/mnemosyne/pixmaps/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionFileNew">
   <property name="icon">
    <iconset>
     <normaloff>../../pixmaps/filenew.png</normaloff>../../pixmaps/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/fileopen.png</normaloff>:/mnemosyne/pixmaps/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/filesave.png</normaloff>:/mnemosyne/pixmaps/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/filesaveas.png</normaloff>:/mnemosyne/pixmaps/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
  </action>
  <action name="actionGettingStarted">
   <property name="text">
    <string>&amp;Getting started</string>
   </property>
  </action>
  <action name="actionShowStatistics">
   <property name="icon">
    <iconset resource="mnemosyne.qrc">
     <normaloff>:/mnemosyne/pixmaps/statistics.png</normaloff>:/mnemosyne/pixmaps/statistics.png</iconset>
   </property>
   <property name="text">
    <string>Show s&amp;tatistics...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionTip">
   <property name="text">
    <string>&amp;Tip of the day</string>
   </property>
  </action>
  <action name="actionManagePlugins">
   <property name="text">
    <string>Manage &amp;plugins...</string>
   </property>
   <property name="toolTip">
    <string>Manage plugins</string>
   </property>
  </action>
  <action name="actionManageTypes">
   <property name="text">
    <string>&amp;Manage card types...</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>S&amp;ync...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCompact">
   <property name="text">
    <string>Compact...</string>
   </property>
   <property name="toolTip">
    <string>Compact database</string>
   </property>
  </action>
  <action name="actionFindDuplicates">
   <property name="text">
    <string>Find duplicate questions</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mnemosyne.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAddCards</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>add_cards</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>file_new</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>file_open</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>file_save</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileSaveAs</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>file_save_as</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditCurrentCard</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>edit_current_card</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeleteCurrentCard</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>delete_current_card</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionManagePlugins</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>manage_plugins</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionManageTypes</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>manage_card_types</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowStatistics</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>show_statistics</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>import_file</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>export_file</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConfigure</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>configure</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionActivateCards</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>activate_cards</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSync</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>sync</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBrowseCards</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>browse_cards</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTip</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>show_tip</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGettingStarted</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>show_getting_started</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>show_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCompact</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>compact_database()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFindDuplicates</sender>
   <signal>triggered()</signal>
   <receiver>MainWdgt</receiver>
   <slot>find_duplicates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
